components/GraphDisplays/monthlyExpenses.tsx

```diff
@@ -0,0 +1,228 @@
+import { AntDesign } from "@expo/vector-icons";
+import React, { useEffect, useState } from "react";
+import { View, Text } from "react-native";
+import { TouchableHighlight } from "react-native-gesture-handler";
+import { EventCallbackInterface, StringOrNumberOrList } from "victory-core";
+import { VictoryAxis, VictoryBar, VictoryChart, VictoryTheme } from "victory-native";
```

Comment:
You are using victory-native but it is not in the package.json. This probably indicates you are working with a package.json whose updates were not tracked, or your local node_modules already contains victory-native but your package.json does not (ie. did not run npm install).